config.def.h

```diff
@@ -59,6 +59,9 @@ static const bool ANTI_ALIAS = true;
  */
 static const bool ALPHA_LAYER = false;
 
+/* cache size for imlib2, in bytes */
+static const int CACHE_SIZE = 4 * 1024 * 1024; /* 4MiB */
```

Comment:
```suggestion
static const int CACHE_SIZE = 4 << 20; /* 4MiB */
```

Reply:
@N-R-K do u think this affects readabilty? 
I mean if anything this should perform better then again the operation is done only once so bitshift vs multiplication speed differnce is moot.

Reply:
I was aiming for readability. I don't think the generated object file changed since this is a pretty trivial optimization. I also think there are enough comments to make it clear what the value is in either case and leaned towards the more compact expression.

Reply:
It shouldn't matter performance wise, it will get optimized away. I think it's more clearer what's going on with `* 1024 * 1024` compared to bitshift.